src/main/java/xbony2/huesodewiki/recipe/recipes/CraftingRecipe.java

```diff
@@ -123,8 +123,8 @@ public String getRecipes(ItemStack itemstack){
 					ShapedOreRecipe shapedrecipe = (ShapedOreRecipe)recipe;
 					ret.append("{{Cg/Crafting Table\n");
 					
-					int maxHeight = shapedrecipe.getHeight();
-					int maxWidth = shapedrecipe.getWidth();
+					int maxHeight = shapedrecipe.getRecipeHeight();
+					int maxWidth = shapedrecipe.getRecipeWidth();
```

Comment:
I assume this is removing a depreciation.

Reply:
Yes, that's exactly that. They call getHeight/getWidth anyways.

Reply:
swell